code/style.css

```diff
@@ -1,3 +1,68 @@
+* {
+  box-sizing: border-box;
+  text-align: center;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #346751;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  color: #ffffff;
+  margin: 0;
+}
+
+header {
+  text-align: center;
+  position: absolute;
+  background-color: #161616;
+  margin-bottom: 30px;
+  width: 100%;
+}
+
+img {
+  border-radius: 50%;
+  border: 2px solid #161616;
```

Comment:
Maybe it will look more responsive if the image size is changed depending on the device size?